packages/typescript/src/exporters/mlflow.ts

```diff
@@ -137,6 +138,10 @@ export class MlflowSpanProcessor implements SpanProcessor {
 }
 
 export class MlflowSpanExporter implements SpanExporter {
+  constructor(private hostConfig: MLflowTracingConfig) {
+    this.hostConfig = hostConfig;
```

Comment:
Since parameter property is used here, no need to assign manually (https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties)
```suggestion
```